MuonAnalysis/MomentumScaleCalibration/plugins/MuScleFitMuonSelector.cc

```diff
@@ -217,7 +217,9 @@ void MuScleFitMuonSelector::selectMuons(const edm::Event & event, std::vector<Mu
   if( speedup_ == false )
   {
     if( PATmuons_ ) {
```

Comment:
@monttj which one? L217 or L219?
I am planning a new PR in few days with few other fixes for 74X/75X miniAODSIM
so I can remove the line if not needed